crates/node/src/cli/sync.rs

```diff
@@ -66,16 +77,73 @@ pub struct SyncParams {
     /// blocks. This can either be once the node has caught up with the head of
     /// the chain or when it has synced as many blocks as specified by
     /// --n-blocks-to-sync.
-    #[clap(env = "MADARA_STOP_ON_SYNC", long, default_value_t = false)]
+    #[clap(env = "MADARA_STOP_ON_SYNC", long, value_name = "STOP ON SYNC", default_value_t = false)]
```

Comment:
adding value_name to a bool seems weird?

Reply:
Yep this makes sense. Removed.